addons/website_crm_partner_assign/static/src/js/crm_partner_assign.js

```diff
@@ -286,4 +287,15 @@ publicWidget.registry.crmPartnerAssign = publicWidget.Widget.extend({
         }
     },
 });
+
+if ($("#force_opportunities_display").length !== 0 ) {
```

Comment:
I'm of the opinion we shouldn't introduce new jquery code even in code that heavily uses it so
```suggestion
if (document.getElementById("force_opportunities_display")) {
```